examples/simpleplots.py

```diff
@@ -6,27 +6,34 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+# Get the path of the test MCMC chain file
```

Comment:
Is it possible instead of
root_file_path = os.path.join(script_dir, "testchaindata.root")
pass "testchaindata.root" as argument?
and if argument not specified then hardcode "testchaindata.root"?
this would make using it for other purposes slilghy easier